packages/shorebird_cli/test/src/commands/patch/patch_ios_framework_command_test.dart

```diff
@@ -767,7 +767,15 @@ Please re-run the release command for this version or create a new release.'''),
           ),
         ),
       ).called(1);
-      verify(() => logger.success('\n✅ Published Patch!')).called(1);
```

Comment:
Would probably be easier to fix these to just include a patch number, but sure. 🤷

Reply:
They do – this same message still prints (including the patch number), I just moved the code to a shared location (the code push client wrapper)